migrations/1719483706248-CreateProfileTable.ts

```diff
@@ -8,13 +8,17 @@ export class CreateProfileTable1719483706248 implements MigrationInterface {
         await queryRunner.query(`ALTER TABLE "users" ADD "profileId" uuid`);
         await queryRunner.query(`ALTER TABLE "users" ADD CONSTRAINT "UQ_b1bda35cdb9a2c1b777f5541d87" UNIQUE ("profileId")`);
         await queryRunner.query(`ALTER TABLE "users" ADD CONSTRAINT "FK_b1bda35cdb9a2c1b777f5541d87" FOREIGN KEY ("profileId") REFERENCES "profiles"("id") ON DELETE NO ACTION ON UPDATE NO ACTION`);
+        await queryRunner.query(`ALTER TABLE "users" DROP CONSTRAINT "FK_b1bda35cdb9a2c1b777f5541d87"`);
+        await queryRunner.query(`ALTER TABLE "users" ADD CONSTRAINT "FK_b1bda35cdb9a2c1b777f5541d87" FOREIGN KEY ("profileId") REFERENCES "profiles"("id") ON DELETE CASCADE ON UPDATE NO ACTION`);
```

Comment:
But why you cannot do it instead of creating `ON DELETE NO ACTION ON UPDATE NO ACTION` at line 10?